tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -17,58 +17,24 @@
 from fauxfactory import gen_string
 from nailgun import entities
 from pytest import skip
-from robottelo import ssh
-from robottelo.config import settings
 from robottelo.constants import (
     COMPUTE_PROFILE_SMALL,
     FOREMAN_PROVIDERS,
     GCE_EXTERNAL_IP_DEFAULT,
     GCE_MACHINE_TYPE_DEFAULT,
-    GCE_NETWORK_DEFAULT
+    GCE_NETWORK_DEFAULT,
+    GCE_SETTINGS
```

Comment:
I don't see this added in https://github.com/SatelliteQE/robottelo/blob/8c62891b3e880f3461ae9fe8c3a4350956752747/robottelo/constants.py

Reply:
Great Catch, I wonder flake8 didnt catch that. Fixed